File: src/test/java/com/hdjunction/domain/code/CodeGroupTest.java
package com.hdjunction.domain.code;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class CodeGroupTest {

    @Test
    @DisplayName("코드 그룹 join")
    void code_group_join() {
        assertThat(CodeGroup.GENDER.getCodeGroup()).isEqualTo("M: 남, F: 여");
    }
}

File: src/test/java/com/hdjunction/domain/code/CodeGroupTest.kt
package com.hdjunction.domain.code

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

internal class CodeGroupTest {

    @Test
    @DisplayName("코드 그룹 join")
    fun `code group join`() {
        assertThat(CodeGroup.GENDER.getCodeGroups()).isEqualTo("M: 남, F: 여")
    }
}